docs/apache-airflow-providers-google/operators/cloud/looker.rst

```diff
@@ -0,0 +1,93 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+
+Google Cloud Looker Operators
+===============================
+
+Looker is a business intelligence software and big data analytics platform that
+helps you explore, analyze and share real-time business analytics easily.
+
+Looker has a Public API and associated SDK clients in different languages,
+which allow programmatic access to the Looker data platform.
+
+For more information visit `Looker API documentation <https://docs.looker.com/reference/api-and-integration>`_.
+
+Prerequisite Tasks
+------------------
+
+To use these operators, you must do a few things:
+
+* Install API libraries via **pip**.
+
+.. code-block:: bash
+
+  pip install 'apache-airflow[google]'
+
+Detailed information is available for :doc:`Installation <apache-airflow:installation/index>`.
+
+Communication between Airflow and Looker is done via `Looker API <https://docs.looker.com/reference/api-and-integration/api-reference/v4.0>`_.
+To facilitate the API communication Looker operators use `Looker SDK <https://pypi.org/project/looker-sdk/>`_ as an API client.
+Before calling API, Looker SDK needs to authenticate itself using your Looker API credentials.
+
+* Obtain your Looker API credentials using instructions in the `Looker API documentation <https://docs.looker.com/reference/api-and-integration/api-auth#authentication_with_an_sdk>`_.
+
+* Obtain your Looker API path and port as described in the `Looker API documentation <https://docs.looker.com/reference/api-and-integration/api-getting-started#looker_api_path_and_port>`_.
+
+* Setup a Looker connection in Airflow. You can check :doc:`apache-airflow:howto/connection`
+
+See the following example of a connection setup:
```

Comment:
Documentation on connection setup should be in a separate documentation file.
https://airflow.apache.org/docs/apache-airflow-providers-google/stable/connections/index.html
In this documentation, it is worth adding at least one URL configuration example.

Reply:
Got it! I think I was under impression that only connections that have their own `Connection Type` goes there, and our connection type is `HTTP`. Sure, I can create a new connection called something like `Google Cloud Looker Connection` there. Will also add a URL configuration example.

Reply:
Done! ✅